dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-msgraph-dynamic/package.json

```diff
@@ -6,18 +6,25 @@
   "license": "Apache-2.0",
   "private": true,
   "publishConfig": {
-    "access": "public",
-    "main": "dist/index.cjs.js",
-    "types": "dist/index.d.ts"
+    "access": "public"
   },
   "backstage": {
     "role": "backend-plugin-module",
-    "supported-versions": "1.26.5"
+    "supported-versions": "1.26.5",
```

Comment:
This should be 1.27.7.  I think we have to bump up the plugin version that corresponds to this release

Reply:
true but also no one is consuming this information. Anyway, PR updated.... and we can update these values again when we finally get to 1.29